fluent-package/yum/systemd-test/update-from-v4.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/commonvar.sh
+
+# Install v4
+sudo rpm --import https://packages.treasuredata.com/GPG-KEY-td-agent
+case ${distribution} in
+    amazon)
+        cat > td.repo <<'EOF';
+[treasuredata]
+name=TreasureData
+baseurl=https://packages.treasuredata.com/4/amazon/$releasever/$basearch
+gpgcheck=1
+gpgkey=https://packages.treasuredata.com/GPG-KEY-td-agent
+EOF
+        sudo mv td.repo /etc/yum.repos.d/
+        ;;
+    *)
+        cat > td.repo <<'EOF';
+[treasuredata]
+name=TreasureData
+baseurl=https://packages.treasuredata.com/4/redhat/$releasever/$basearch
+gpgcheck=1
+gpgkey=https://packages.treasuredata.com/GPG-KEY-td-agent
+EOF
+        sudo mv td.repo /etc/yum.repos.d/
+        ;;
+esac
+sudo $DNF update -y
+sudo $DNF install -y td-agent-4.5.0-1.*.x86_64
+
+sudo systemctl enable --now td-agent
+systemctl status --no-pager td-agent
+
+# Install the current
+sudo $DNF install -y \
+    /vagrant/${distribution}/${DISTRIBUTION_VERSION}/x86_64/Packages/fluent-package-[0-9]*.rpm
+
+# Test: service status
+sudo systemctl enable --now fluentd
+systemctl status --no-pager fluentd
+systemctl status --no-pager td-agent
+
+# Test: config migration
+test -L /etc/td-agent
+test -e /etc/td-agent/td-agent.conf
+
+# Test: log file migration
+test -L /var/log/td-agent
+test -e /var/log/td-agent/td-agent.log
+
+# Test: environmental variables
+pid=$(eval $(systemctl show fluentd --property=MainPID) && echo $MainPID)
+env_vars=$(sudo sed -e 's/\x0/\n/g' /proc/$pid/environ)
+test $(eval $env_vars && echo $HOME) = "/var/lib/fluent"
+test $(eval $env_vars && echo $LOGNAME) = "fluentd"
+test $(eval $env_vars && echo $USER) = "fluentd"
+test $(eval $env_vars && echo $FLUENT_CONF) = "/etc/fluent/td-agent.conf"
+test $(eval $env_vars && echo $FLUENT_PACKAGE_LOG_FILE) = "/var/log/fluent/td-agent.log"
+test $(eval $env_vars && echo $FLUENT_PLUGIN) = "/etc/fluent/plugin"
+test $(eval $env_vars && echo $FLUENT_SOCKET) = "/var/run/fluent/fluentd.sock"
+
+# Test: logs
+sleep 3
+! grep -q -e '\[warn\]' -e '\[error\]' -e '\[fatal\]' /var/log/fluent/td-agent.log
```

Comment:
I noticed this should fail because the Fluentd works with the old config, which should output a warning log...

Why is this not failing...

Reply:
I'm testing longer sleep on my fork.

Reply:
I will remove this test from `update-from-v4` test after I find the cause.

Reply:
https://github.com/fluent/fluent-package-builder/pull/530#discussion_r1268012097